examples/express-sample/app.js

```diff
@@ -23,8 +24,7 @@ app.set("view engine", "ejs");
 // Add the Raygun breadcrumb Express handler
 app.use(raygunClient.expressHandlerBreadcrumbs);
 
-// uncomment after placing your favicon in /public
-// app.use(favicon(__dirname + '/public/favicon.ico'));
+app.use(favicon(__dirname + "/public/favicon.ico"));
```

Comment:
Will it throw an error if no favicon is present?

Reply:
A favicon has been included in the repo